docs/product-assembly_running_employee_rostering_in_wb/src/main/asciidoc/wb-employee-rostering-balance-employees-shift-rule-proc.adoc

```diff
@@ -0,0 +1,32 @@
+[id='wb-employee-rostering-balance-employees-shift-rule-proc']
+= Create the `BalanceEmployeesShiftNumber` guided rule 
+
+The `BalanceEmployeesShiftNumber` guided rule creates a soft constraint that ensures all employees are assigned a shift, or that all employees are assigned an equal number of shifts. In the employee rostering example, this constraint is created using the guided rule editor. 
+
+image::BalanceEmployeesShiftNumber.png[BalanceEmployeesShiftNumber Guided Rule]
+
+.Procedure
+. Go to *Menu* -> *Design* -> *Projects* and click the project name.
+. Click *Create New Asset* -> *Guided Rule*.
+. Enter `BalanceEmployeesShiftNumber` as the *Guided Rule* name and select the `employeerostering.employeerostering` *Package*. 
+. Click *Ok* to create the rule asset.
+. Add a *WHEN* condition by clicking the image:5686.png[] in the *WHEN* field.
+. Select `Employee` in the *Add a condition to the rule...* window. Click *+Ok*. 
+. Click on the `Employee` condition to modify the constraints and add the variable name `$employee`.
+. Add the *WHEN* condition `From Accumulate...`. 
```

Comment:
Per IBM SG, don't retain the trailing ellipsis when referring to UI text that has it at the end. So should be "the **Add a condition to the rule** window", "condition `From Accumulate`", etc. The only exception I've found is when the ellipsis is not the final element, such as "click **(show options...)**", at the bottom of the designer. I'd search all ellipses and see if they are per the UI and can be removed. Super minor, but just sayin'. Up to you.